### PR TITLE
Changes to CLI and 2 fixes, fix response handling in get fn, fix loading json data, remove file logging

### PR DESCRIPTION
**Pull request by roks0n**
**CLI changes:**
- removed `snapLogging` and `restoreLogging`, my thinking is we shouldn't modify/hide handlers. Let's show all errors in the CLI and work towards fixing them if they occur.
- modified the DATA object so we don't create our own `__setattr__` method which is a class method that is called by `setattr` builtin method, instead, we can check and call `set()` on daemon attribute in a bit more pythonic way
- changed the `__repr__` which is a minimal thing, nothing major
- added a check if folder exists before creating it

**REST changes:**
Fixed response handling in the get function. Before this fixed I got an error when calling `delegate` in the CLI, eg:
```
File "arky/arky/rest.py", line 64, in get
    data = data[return_key]
KeyError: 'delegate'
```

**OTHER changes**
I've also done some changes in utils.py which fixed the following error I got when calling `account status`:
```
File "arky/arky/cli/account.py", line 192, in status
  File "arky/arky/util.py", line 205, in prettyPrint
    sys.stdout.write(pretty)
write() argument must be str, not bytes
```

I'm trying to work on tests for the CLI, but every time I want to start I find something I just need to modify/fix. :)

### CODE REVIEW

File: arky/cli/__init__.py [hunk 1]
```diff
@@ -51,31 +51,37 @@ def _whereami():
...
+		Clears all defined attribute. This is called when user switches network
+		"""
+		self.delegate = {}
+		self.ledger = {}
+		self.account = {}
+		self.firstkeys = {}
+		self.secondkeys = {}
+		self.executemode = False
+		self.escrowed = False
+		if self.daemon:
```
Comment (roks0n) [inline arky/cli/__init__.py]: I'm not sure if I should add a validator to check if value really is of type `threading.Event` or is this not necessary? 🤔

**Comment by Moustikitos** [comment]
`743ad50`: NOT OKAY - CLI uses logging to write command history. The `snapLogging` and `restoreLogging` work perfectly when you enter in and leave out CLI. Plus, error are prompted on the console when it occurs.

`67786d6` : OKAY

`4da9f89` : OKAY

`6469abd` : PARTIALY OKAY - you have to leave the encoding thing because some errors occur on windows console.

`8309cdf` : OKAY

`435f85f` : NOT OKAY - For now, do not touch to `Data` object. Plus, no tabs in `docstring` to be correctly handled by `docopt` package :)

**Comment by roks0n** [comment]
`743ad50`: curious and would like to understand, why do we need to log command history?

I'll change not okay's. :)

**Comment by Moustikitos** [comment]
command history is usefull when you want to check things you've done elswhere than devnet :)

**Comment by roks0n** [comment]
@Moustikitos I've reverted some of the changes.

**Comment by Moustikitos** [comment]
Okay, sounds good !

**Pull request merged**